src/client/app/containers/admin/AddMetersContainer.ts

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { connect } from 'react-redux';
+import AddMetersComponent from '../../components/admin/AddMetersComponent';
+import { fetchMetersDetailsIfNeeded } from '../../actions/meters';
+import { Dispatch } from '../../types/redux/actions';
+
+function mapDispatchToProps(dispatch: Dispatch) {
+	return {
+		fetchMeterDetailsIfNeeded: (alwaysFetch?: boolean) => dispatch(fetchMetersDetailsIfNeeded(alwaysFetch))
```

Comment:
You always pass true here (because you're always forcing it) so it might be cleaner to return `{ fetchMeterDetails: () => dispatch(fetchMeterDetailsIfNeeded(true)) }`

Reply:
Yes, I will do that. That makes sense.